### PR TITLE
Add Prompt to ProvidedOktaConfigModel

### PR DESCRIPTION
**Pull request by reldredge71**
`prompt` parameter allows us to control the behavior when a user is directed to Okta.

https://developer.okta.com/docs/reference/api/oidc/#parameter-details

### CODE REVIEW

File: Example/Example/App/Authentication/AuthenticationView.swift [hunk 1]
```diff
@@ -23,7 +23,8 @@ class AuthenticationView: UIViewController {
             logoutRedirectUri: "",
             issuer: "",
             redirectUri: "",
-            scopes: "openid profile offline_access email"
+            scopes: "openid profile offline_access email",
+            prompt: nil,
```
Comment (wrandall22) [inline Example/Example/App/Authentication/AuthenticationView.swift]: I wonder if we should just default to `login` instead?

File: Sources/OktaAuthentication/ProvidedOktaConfigModel.swift [hunk 2]
```diff
@@ -15,14 +15,16 @@ public struct ProvidedOktaConfigModel: OktaConfigModelType, MinimumOktaConfigAtt
...
     
-    public init(clientId: String, logoutRedirectUri: String, issuer: String, redirectUri: String, scopes: String) {
+    public init(clientId: String, logoutRedirectUri: String, issuer: String, redirectUri: String, scopes: String, prompt: String?) {
         
         self.clientId = clientId
         self.logoutRedirectUri = logoutRedirectUri
         self.issuer = issuer
         self.redirectUri = redirectUri
-        self.scopes = scopes
+        self.scopes = scopes'
```
Comment (frett) [inline Sources/OktaAuthentication/ProvidedOktaConfigModel.swift]: I don't think you meant to add a `'` to this line

File: Example/Example/App/Authentication/AuthenticationView.swift (hunk 1 above)
Comment (frett) [reply Example/Example/App/Authentication/AuthenticationView.swift]: yeah, my feeling for mobile is to default to `login`

**Review by wrandall22 — APPROVED** [review]

**Comment by wrandall22** [comment]
Does a version need to be bumped?

**Review by frett — APPROVED** [review]

**Comment by frett** [comment]
it looks like a new version tag needs to be created after this is merged

**Comment by frett** [comment]
SPM uses a reference to the the git repo to pull in a dependency, so tag the new commit and update the reference in mpdx-ios.

**Pull request merged**